ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Entity.java

```diff
@@ -0,0 +1,31 @@
+package com.github.migi_1.Context.model.entity;
+
+/**
+ * Superclass of all object that implement some MovableBehaviour.
+ *
+ * @author Marcel
+ *
+ */
+public abstract class Entity {
+
+    /**
+     * Move behaviour of particular Entity.
+     */
+    protected MoveBehaviour moveBehaviour;
```

Comment:
Because there is a getter and a setter for this moveBehaviour, you can just make it private.